### ISSUE TITLE
Docs: Add link to deployed Jupyter Book in README and in the project sidebar

### ISSUE THREAD

**Issue by matthewfeickert** [hsf-training/deep-learning-intro-for-hep#4]
It would be great if from the README there was a link to the deployed build of the Jupyter Book. Additionally if that URL could get added as the "website" metadata in the "repository details" section on the sidebar of the landing page that would be great.

**Comment by jpivarski** [comment]
I need to get back to finish writing this. (It's not even remotely done.)

**Comment by matthewfeickert** [comment]
Yeah, this wasn't intended as a "why isn't this done" ping. More that people like me would be interested to see this in the "getting developed" stages as you build it.

**Comment by jpivarski** [comment]
Done: I've now verified that the GitHub Pages are being built correctly and I've included a link from the README to the first page.

So far, the converted part includes `lesson-1-introduction` from the [CoDaS-HEP lectures](https://github.com/jpivarski-talks/2024-07-24-codas-hep-ml). `lesson-2-issues` will be a large part of what remains, and `lesson-3-project` is just a project description. I don't know what/how much of `lesson-4-architectures` to do, but I'll make a decision about that when I get to it.
